src/packagedetails.cpp

```diff
@@ -42,7 +42,7 @@ PackageDetails::PackageDetails(OCI::Package *package,int count, int packagecount
 
     QObject::connect( meta, SIGNAL( finished( QString,QString ) ), this, SLOT( dataChanged( QString,QString ) ) );
 
-    m_showDescription = new QLabel( QString( "<a href = %1>Show Details</a>" ).arg( count ) );
+     m_showDescription = new QLabel( i18n("<a href = %1 style=\"text-decoration:none;\">Show Details</a>", count) );
```

Comment:
It is mentioned in the individual commit but not in the PR. I shall make the necessary changes